### PR TITLE
Add HTML AAM considerations to PR template and enable for HTML

### PR DESCRIPTION
**Pull request by annevk**
@domenic @scottaohara let me know if you have any thoughts. The main thing this adds to the PR template for HTML will look like this:

- [ ] Corresponding [HTML AAM](https://w3c.github.io/html-aam/) issue or PR:

and I would expect it to be completed like this:

- [x] Corresponding [HTML AAM](https://w3c.github.io/html-aam/) issue or PR: N/A

or:

- [x] Corresponding [HTML AAM](https://w3c.github.io/html-aam/) issue or PR: PR link

(This has been edited to instead feature the latest suggestion.)

### CODE REVIEW

**Comment by aphillips** [comment]
What about other horizontal concerns, e.g. security, privacy, sustainability, and the like?

Why would separate PRs be created for a11y/i18n? Shouldn't these types of issues be addressed in PR being created with the template? 

I might prefer to just see the `i18n-tracker` label on PRs that have I18N-related gorp in them. If the checkbox is just a reminder that we think I18N is important, perhaps instead:

- [ ] Horizontal issues (A11Y, I18N, etc.) have been addressed

Or am I not understanding?

**Comment by annevk** [comment]
Hmm yeah.

Maybe I should restrict this to accessibility and in particular [HTML AAM](https://w3c.github.io/html-aam/) which "frequently" ends up needing changes due to changes in HTML semantics.

Most other horizontal concerns don't really have that kind of integration.

**Comment by scottaohara** [comment]
thanks for this @annevk 

i think this would be a good add.  arguably a second check for aria in html issue/pr might be helpful?  but, maybe just html aam is enough since that'll get it on the ARIA wg's radar and then we can determine if an aria in html issue/pr would be necessary or not.

**Review by domenic — APPROVED** [review]

File: factory.py [hunk 1]
```diff
@@ -80,6 +80,11 @@ def fill_template(contents, variables):
             for entry in data:
                 output += "\n   * {}: …".format(entry)
             data = output
+        elif variable == "a11y-check":
+            output = ""
+            if data:
+                output = "\n- [ ] Corresponding [HTML AAM](https://w3c.github.io/html-aam/) & [WAI-ARIA](https://w3c.github.io/aria/) issues & PRs:"
```
Comment (zcorpan) [inline factory.py:86]: @scottaohara asked to add "ARIA in HTML" which is https://w3c.github.io/html-aria/ . I think ARIA usually don't need changes for changes in HTML.

Comment (annevk) [reply factory.py:86]: Sigh. Okay. Will create some more PRs.

**Pull request merged**